### PR TITLE
min. software requirements

### PR DESCRIPTION
**Pull request by MadinaB**
Running everything with sudo, having mercurial, python >=3.6 and pip for python >=3.6  are minimum requirements essential for the project :) 
Vagrantfile is provided for easy project set up and configuration.
#171 issue is solved here and recommendation regarding #169 (just setting to prev version) is added to readme of /easy-environment-setup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 
 A crawler to find websites that exercise code in Firefox that is not covered by unit tests
 
+## Software requirements
+
+- You will need python>=3.5, pip for python>=3.5 and [Mercurial](https://www.mercurial-scm.org/) source control management tools installed on your machine to run the script.
```
Comment (rhcu) [inline README.md]: Hi! You need to use Python 3.6+.

Comment (MadinaB) [reply README.md]: Thank you for review! Will update that asap :)

Comment (MadinaB) [reply README.md]: Updated

File: README.md [hunk 2]
```diff
@@ -3,6 +3,12 @@
 
 A crawler to find websites that exercise code in Firefox that is not covered by unit tests
 
+## Software requirements
+
+- You will need python>=3.6, pip for python>=3.6 and [Mercurial](https://www.mercurial-scm.org/) source control management tools installed on your machine to run the script.
```
Comment (marco-c) [inline README.md]: Just list the requirements, no need for the longer description.
```
- Python 3.6+
- [Mercurial](https://www.mercurial-scm.org/)
```

Not needed to specify `pip` as it's basically implied.

File: easy-environment-setup/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+sudo pip install -r test-requirements.txt
+```
+
+2. Download artifacts:
+- Try:
+```
+python download_artifacts.py
+```
+
+If you are facing `TaskclusterFailure: rootUrl option is required`, just do following:
```
[the remark this answers is not in the document]
Comment (MadinaB) [reply easy-environment-setup/README.md]: Yep

File: README.md (hunk 2 above)
Comment (MadinaB) [reply README.md]: Yep

**Pull request closed without merging**